app/code/Magento/Contact/Api/MailInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * Contact module base controller
+ */
+namespace Magento\Contact\Api;
+
+interface MailInterface
+{
+    /**
+     * Send email from contact form
+     *
+     * @param string $replyTo Reply-to email address
+     * @param array $variables Email template variables
```

Comment:
I now understand that these rules only apply to service contracts in the sense of interfaces that can be exposed as web API, so I assume that I can leave the signature as it is, after moving the interfaces out of `/Api/`